### PR TITLE
Terraform to perform CRUD operations for LB-POOL resource

### PR DESCRIPTION
**Pull request by Pushpalatha555**
**VALIDATED THE LB-POOL FOR CREATE/DELETE/UPDATE**
CREATE
CREATE-UI
UPDATE
UPDATE-UI
DELETE
DELETE-UI

### CODE REVIEW

File: examples/data-sources/hpegl_vmaas_pool_member_group/data-source.tf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# (C) Copyright 2021 Hewlett Packard Enterprise Development LP
+
+data "hpegl_vmaas_pool_member_group" "tf_pool_group" {
```
Comment (manjunath-batakurki) [inline examples/data-sources/hpegl_vmaas_pool_member_group/data-source.tf:3]: Can you name it better ? it should not be understand as data source for Group. Name it as hpegl_vmaas_lb_pool_member_group

File: examples/data-sources/hpegl_vmaas_passive_monitor/data-source.tf [hunk 2]
```diff
@@ -0,0 +1,6 @@
+ (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+
+data "hpegl_vmaas_passive_monitor" "tf_lb_passive" {
```
Comment (manjunath-batakurki) [inline examples/data-sources/hpegl_vmaas_passive_monitor/data-source.tf:3]: u have one resource for monitor. why multiple data sources for monitor ?

File: internal/cmp/data_source_active_monitor.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
+// (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+
+package cmp
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/HewlettPackard/hpegl-vmaas-cmp-go-sdk/pkg/client"
```
Comment (manjunath-batakurki) [inline internal/cmp/data_source_active_monitor.go:10]: Please note that for every resource, there should be one data source definition. We cant have multiple of them. What is difference this one and the other one ?

File: internal/cmp/data_source_pool_member_group.go [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+	"github.com/HewlettPackard/hpegl-vmaas-cmp-go-sdk/pkg/client"
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/internal/utils"
+	"github.com/tshihad/tftags"
+)
+
+type poolMemberGroupds struct {
+	lbClient *client.LoadBalancerAPIService
+}
+
+func newPoolMemberGroupDS(poolMemberGroupClient *client.LoadBalancerAPIService) *poolMemberGroupds {
```
Comment (manjunath-batakurki) [inline internal/cmp/data_source_pool_member_group.go:19]: poolMemberGroups

Comment (manjunath-batakurki) [inline internal/cmp/data_source_pool_member_group.go:19]: use LB prefix, newLBPoolMemberGroupDS

File: internal/resources/constants.go [hunk 5]
```diff
@@ -35,6 +38,9 @@ const (
 	ResNetwork                    = "hpegl_vmaas_network"
 	ResRouter                     = "hpegl_vmaas_router"
 	ResLoadBalancer               = "hpegl_vmaas_load_balancer"
+	ResActiveMonitor              = "hpegl_vmaas_active_monitor"
+	ResPoolMemeberGroup           = "hpegl_vmaas_pool_member_group"
+	ResPassiveMonitor             = "hpegl_vmaas_passive_monitor"
```
Comment (manjunath-batakurki) [inline internal/resources/constants.go:43]: are these resources as well ? why are u adding these here ?

File: internal/resources/data_source_lb_passive_monitors.go [hunk 6]
```diff
@@ -0,0 +1,53 @@
+// (C) Copyright 2021 Hewlett Packard Enterprise Development LP
+
+package resources
+
+import (
+	"context"
```
Comment (manjunath-batakurki) [inline internal/resources/data_source_lb_passive_monitors.go:6]: dup file, not needed.

File: internal/resources/data_source_lb_pool_member_group.go [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+import (
+	"context"
+
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/internal/utils"
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/pkg/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func PoolMemeberGroupData() *schema.Resource {
```
Comment (manjunath-batakurki) [inline internal/resources/data_source_lb_pool_member_group.go:14]: add LB as prefix

File: examples/data-sources/hpegl_vmaas_pool_member_group/data-source.tf (hunk 1 above)
Comment (Pushpalatha555) [reply examples/data-sources/hpegl_vmaas_pool_member_group/data-source.tf:3]: done

File: internal/cmp/data_source_pool_member_group.go (hunk 4 above)
Comment (Pushpalatha555) [reply internal/cmp/data_source_pool_member_group.go:19]: done

File: internal/resources/constants.go (hunk 5 above)
Comment (Pushpalatha555) [reply internal/resources/constants.go:43]: removed

File: internal/resources/data_source_lb_pool_member_group.go (hunk 7 above)
Comment (Pushpalatha555) [reply internal/resources/data_source_lb_pool_member_group.go:14]: done

File: examples/data-sources/hpegl_vmaas_passive_monitor/data-source.tf (hunk 2 above)
Comment (Pushpalatha555) [reply examples/data-sources/hpegl_vmaas_passive_monitor/data-source.tf]: removed

File: internal/cmp/data_source_active_monitor.go (hunk 3 above)
Comment (Pushpalatha555) [reply internal/cmp/data_source_active_monitor.go]: updated

File: internal/resources/data_source_lb_passive_monitors.go (hunk 6 above)
Comment (Pushpalatha555) [reply internal/resources/data_source_lb_passive_monitors.go]: removed

**Review by manjunath-batakurki — APPROVED** [review]
LGTM

**Review by gandharvas — APPROVED** [review]
@Pushpalatha555 Can you pelase fix the CI in your next PR please

**Pull request merged**